packages/checkout/widgets-lib/src/resources/text/textConfig.ts

```diff
@@ -67,6 +67,10 @@ export const text = {
       body: ['You can try again or contact', 'support', 'for help.'],
       actionText: 'Try again',
     },
+    [SharedViews.SERVICE_UNAVAILABLE_ERROR_VIEW]: {
+      heading: (service: string) => `${service} not available in your region`,
```

Comment:
@sharonsheah @deepti-imx 
This solution means we need to hardcode a string in the props for the view, which also gets passed straight into the rendered text output. I think this what we are trying to get away from with the textConfig in general, so that we can define the text as constant. 
Perhaps you could pass a enum prop into the view which is sued to choose the specific constant text to be rendered? For example:
[SharedViews.SERVICE_UNAVAILABLE_ERROR_VIEW]: {
      swapHeading: 'Swapping is not available in your region',
      bridgeHeading: 'Bridging is not available in your region'
      onrampHeading: ...
 
This also means that we can change each of the service headings specifically for swap / bridge / other service, if we need to without having to decouple from `${service} is not available...`